src/native/CMakeLists.txt

```diff
@@ -11,7 +11,12 @@ cmake_minimum_required(VERSION 3.0.0)
 project (ecos)
 set(PROJECT_VERSION_MAJOR 0)
 set(PROJECT_VERSION_MINOR 0)
-set(PROJECT_VERSION_PATCH 1)
+set(PROJECT_VERSION_PATCH 8)
```

Comment:
Does this version do anything? Should it be bumped along with the version in sbt? Should it be bumped when this file is changed?

Reply:
I have no idea. I have never touched it in the previous bumps.